### PR TITLE
test(numeraldate,checkbox,dateinput,daterange): update tests with a new approach

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

Apply new `url` approach in tests for:
`Design System:`
- Numeral Date (`timing`: was `1:42` -> `00:30`).

`Experimental:`
- Checkbox (`timing`: was `3:19` -> `1:05`).
- Date Input (`timing`: was `5:26` -> `2:01`).
- Date Range (`timing`: was `1:13` -> `00:30`).

### Current behaviour

Currently we are passing all parameters via `knobs`. 

### Checklist

- [ ] Screenshots are included in the PR
- [ ] Carbon implementation and Design System documentation are congruent
- [ ] All themes are supported
- [x] Commits follow our style guide
- [ ] Unit tests added or updated
- [x] Cypress automation tests added or updated
- [ ] Storybook added or updated
- [ ] Typescript `d.ts` file added or updated

### CODE REVIEW

File: cypress/features/regression/experimental/checkbox.feature [hunk 1]
```diff
@@ -1,88 +1,84 @@
...
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | label                   | nameOfObject          |
+      | mp150ú¿¡üßä             | labelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | labelSpecialCharacter |
 
   @positive
   Scenario: Disable and enable checkbox
-    Given I check disabled checkbox
-    When I uncheck disabled checkbox
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "enabledCheckbox" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:15]: could you make change and use naming like `knobName+Parameter`, we have dedicated `json` for every component so `disabledFalse` will be enough.
```suggestion
    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "disabledFalse" object name
```

File: cypress/fixtures/experimental/checkbox.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{
+  "labelOtherLanguage": {
+    "label": "mp150ú¿¡üßä"
+  },
+  "labelSpecialCharacter": {
+    "label": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "enabledCheckbox": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/checkbox.json:8]:
```suggestion
  "disabledFalse": {
```

File: cypress/fixtures/experimental/checkbox.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    "disabled": false
+  },
+  "disabledCheckbox": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/checkbox.json:11]:
```suggestion
  "disabled": {
```

File: cypress/features/regression/experimental/checkbox.feature [hunk 4]
```diff
@@ -1,88 +1,84 @@
...
     Then Checkbox is enabled
 
   @positive
   Scenario: Disable checkbox
-    When I check disabled checkbox
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "disabledCheckbox" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:20]:
```suggestion
    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "disabled" object name
```

File: cypress/features/regression/experimental/checkbox.feature [hunk 5]
```diff
@@ -1,88 +1,84 @@
...
   Scenario: Enable and disable fieldHelpInline
-    Given I check fieldHelpInline checkbox
-    When I uncheck fieldHelpInline checkbox
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "fieldHelpInlineEnabled" object name
     Then Checkbox is not set to fieldHelpInline and has margin set to "0px 0px 0px 16px"
 
   @positive
   Scenario: Enable reverse checkbox
-    When I check reverse checkbox
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "reverseTrue" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:44]:
```suggestion
    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "reverse" object name
```

File: cypress/fixtures/experimental/checkbox.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+  "fieldHelpSpecialCharacter": {
+    "fieldHelp": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "fieldHelpInlineEanabled": {
+    "fieldHelpInline": true
+  },
+  "fieldHelpInlineDisabled": {
+    "fieldHelpInline": true
+  },
+  "reverseTrue": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/checkbox.json:26]:
```suggestion
  "reverse": {
```

File: cypress/fixtures/experimental/checkbox.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+  "disabledCheckbox": {
+    "disabled": true
+  },
+  "fieldHelpOtherLanguage": {
+    "fieldHelp": "mp150ú¿¡üßä"
+  },
+  "fieldHelpSpecialCharacter": {
+    "fieldHelp": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "fieldHelpInlineEanabled": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/checkbox.json:20]:
```suggestion
  "fieldHelpInline": {
```

File: cypress/features/regression/experimental/checkbox.feature [hunk 8]
```diff
@@ -1,88 +1,84 @@
...
-    When I check fieldHelpInline checkbox
-    Then Checkbox is set to fieldHelpInline and has margin-left set to "16px"
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "fieldHelpInlineDisabled" object name
+    Then Checkbox is set to fieldHelpInline and has margin-left set to "0px"
 
   @positive
   Scenario: Enable and disable fieldHelpInline
-    Given I check fieldHelpInline checkbox
-    When I uncheck fieldHelpInline checkbox
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "fieldHelpInlineEnabled" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:39]:
```suggestion
    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "fieldHelpInline" object name
```

File: cypress/features/regression/experimental/checkbox.feature [hunk 9]
```diff
@@ -1,88 +1,84 @@
...
   Scenario Outline: Change Checkbox label align to <direction>
-    When I select labelAlign to "<direction>"
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "<nameOfObject>" object name
     Then Checkbox labelAlign on preview is set to "<direction>"
     Examples:
-      | direction |
-      | left      |
-      | right     |
+      | direction | nameOfObject   |
+      | left      | directionLeft  |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:68]:
```suggestion
      | left      | labelAlignLeft  |
```

File: cypress/features/regression/experimental/checkbox.feature [hunk 10]
```diff
@@ -1,88 +1,84 @@
...
+      | right     | directionRight |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:69]:
```suggestion
      | right     | labelAlignRight |
```

File: cypress/fixtures/experimental/checkbox.json [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+  "labelHelpSpecialCharacter": {
+    "labelHelp": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "smallSize": {
+    "size": "small"
+  },
+  "largeSize": {
+    "size": "large"
+  },
+  "directionLeft": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/checkbox.json:44]:
```suggestion
  "labelAlignLeft": {
```

File: cypress/fixtures/experimental/checkbox.json [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+    "labelAlign": "left"
+  },
+  "directionRight": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/checkbox.json:47]:
```suggestion
  "labelAlignRight": {
```

File: cypress/features/regression/experimental/dateInput.feature [hunk 13]
```diff
@@ -1,130 +1,120 @@
...
   Scenario: Disable Date Input
-    When I disable DateInput component
+    When I open default "Experimental Date Input" component in noIFrame with "dateInput" json from "experimental" using "disabled" object name
     Then Date input is disabled
 
   @negative
   Scenario: Disable and enable Date Input
-    When I disable DateInput component
-      And I enable DateInput component
+    When I open default "Experimental Date Input" component in noIFrame with "dateInput" json from "experimental" using "enabled" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInput.feature:11]:
```suggestion
    When I open default "Experimental Date Input" component in noIFrame with "dateInput" json from "experimental" using "disabledFalse" object name
```

File: cypress/features/regression/experimental/dateInput.feature [hunk 14]
```diff
@@ -1,130 +1,120 @@
...
-      And I select labelAlign to "<labelAlign>"
-    Then label align on preview is set to "<labelAlign>" in IFrame
+    When I open default "Experimental Date Input" component in noIFrame with "dateInput" json from "experimental" using "<nameOfObject>" object name
+    Then label align on preview is set to "<labelAlign>"
     Examples:
-      | labelAlign |
-      | left       |
-      | right      |
+      | labelAlign | nameOfObject   |
+      | left       | directionLeft  |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInput.feature:53]:
```suggestion
      | left       | labelAlignLeft  |
```

File: cypress/features/regression/experimental/dateInput.feature [hunk 15]
```diff
@@ -1,130 +1,120 @@
...
+      | right      | directionRight |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateInput.feature:54]:
```suggestion
      | right      | labelAlignRight |
```

File: cypress/features/regression/experimental/dateRange.feature [hunk 16]
```diff
@@ -1,42 +1,30 @@
...
+      | label                   | nameOfObject             |
+      | mp150ú¿¡üßä             | endLabelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | endLabelSpecialCharacter |
 
   @positive
   Scenario: Enable labels inline checkbox
-    When I set startLabel to "label"
-      And I set endLabel to "label"
-      And I check labelsInline checkbox
+    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "enableLabelInput" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateRange.feature:24]:
```suggestion
    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "labelInline" object name
```

File: cypress/features/regression/experimental/dateRange.feature [hunk 17]
```diff
@@ -1,42 +1,30 @@
...
     Then labels are set to inline
 
   @positive
   Scenario: Enable and disable labels inline checkbox
-    When I set startLabel to "label"
-      And I set endLabel to "label"
-      And I check labelsInline checkbox
-      And I uncheck labelsInline checkbox
+    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "disableLabelInput" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateRange.feature:29]:
```suggestion
    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "labelInlineFalse" object name
```

File: cypress/fixtures/experimental/dateInput.json [hunk 18]
```diff
@@ -0,0 +1,72 @@
+{
+  "disabled": {
+    "disabled": true
+  },
+  "enabled": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateInput.json:5]:
```suggestion
  "disabledFalse": {
```

File: cypress/fixtures/experimental/dateInput.json [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+    "label": "mp150ú¿¡üßä"
+  },
+  "labelSpecialCharacter": {
+    "label": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "labelInline": {
+    "label": "label",
+    "labelInline": true
+  },
+  "directionLeft": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateInput.json:30]:
```suggestion
  "labelAlignLeft": {
```

File: cypress/fixtures/experimental/dateInput.json [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+    "labelAlign": "left",
+    "label": "label",
+    "labelHelp": "labelHelp",
+    "labelInline": true
+  },
+  "directionRight": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateInput.json:36]:
```suggestion
  "labelAlignRight": {
```

File: cypress/fixtures/experimental/dateInput.json [hunk 21]
```diff
@@ -0,0 +1,72 @@
...
+    "label": "label",
+    "labelInline": true,
+    "labelWidth": 100
+  },
+  "sliderWidth0": {
+    "label": "label",
+    "labelInline": true,
+    "labelWidth": 0
+  },
+  "sliderWidth10": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateInput.json:62]: what is the difference between this object and `labelWidth10` above ? Seems to me the same parameters

File: cypress/fixtures/experimental/dateRange.json [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+  "startLabelSpecialCharacter": {
+    "startLabel": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "endLabelOtherLanguage": {
+    "endLabel": "mp150ú¿¡üßä"
+  },
+  "endLabelSpecialCharacter": {
+    "endLabel": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "enableLabelInput": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateRange.json:14]:
```suggestion
  "labelInline": {
```

File: cypress/fixtures/experimental/dateRange.json [hunk 23]
```diff
@@ -0,0 +1,24 @@
...
+    "startLabel": "label",
+    "endLabel": "label",
+    "labelsInline": true
+  },
+  "disableLabelInput": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateRange.json:19]:
```suggestion
  "labelInlineFalse": {
```

File: cypress/locators/date-input/index.js [hunk 24]
```diff
@@ -11,6 +11,7 @@ export const maxDate = () => cy.get(MAX_DATE);
 
 // component preview locators
 export const dateInput = () => cy.iFrame(DATE_INPUT);
```
Comment (DlgSHi) [inline cypress/locators/date-input/index.js:13]: do we need those other `inIFrame` locators ?

File: cypress/support/step-definitions/checkbox-steps.js [hunk 25]
```diff
@@ -1,63 +1,49 @@
 import {
-  checkboxCommonInputField, checkboxLabelPreview,
-  checkbox, checkboxDataComponent, checkboxRole, checkboxByID, dataComponentGroup,
-  labelForIconInCheckboxGroup,
+  checkbox, checkboxRole,
```
Comment (DlgSHi) [inline cypress/support/step-definitions/checkbox-steps.js:2]:
```suggestion
  checkbox,
  checkboxRole,
```

File: cypress/support/step-definitions/date-input-steps.js [hunk 26]
```diff
@@ -11,22 +11,22 @@ const TODAY_KNOBS = Cypress.moment().format('YYYY-MM-DD');
 const TODAY_DATE_INPUT = Cypress.moment().format('DD/MM/YYYY');
 
 Then('Date input is disabled', () => {
-  dateInput().parent()
+  dateInputNoIFrame().parent()
```
Comment (DlgSHi) [inline cypress/support/step-definitions/date-input-steps.js:14]: could we move `parent()` to index file, if we are not using simple `dateInputNoIFrame()` without it

File: cypress/fixtures/experimental/dateRange.json (hunk 23 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateRange.json]: fixed

File: cypress/locators/date-input/index.js (hunk 24 above)
Comment (KatarzynaQA) [reply cypress/locators/date-input/index.js:13]: yes, we need it. We have TC which is opened in Iframe state and use it

File: cypress/support/step-definitions/checkbox-steps.js (hunk 25 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/checkbox-steps.js]: fixed

File: cypress/fixtures/experimental/dateRange.json (hunk 22 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateRange.json]: fixed

File: cypress/fixtures/experimental/dateInput.json (hunk 21 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateInput.json]: removed

File: cypress/fixtures/experimental/dateInput.json (hunk 20 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateInput.json]: fixed

File: cypress/fixtures/experimental/dateInput.json (hunk 19 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateInput.json]: fixed

File: cypress/fixtures/experimental/dateInput.json (hunk 18 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateInput.json]: fixed

File: cypress/features/regression/experimental/dateRange.feature (hunk 17 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateRange.feature]: fixed

File: cypress/features/regression/experimental/dateRange.feature (hunk 16 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateRange.feature]: fixed

File: cypress/features/regression/experimental/dateInput.feature (hunk 15 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateInput.feature]: fixed

File: cypress/features/regression/experimental/dateInput.feature (hunk 14 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateInput.feature]: fixed

File: cypress/features/regression/experimental/dateInput.feature (hunk 13 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateInput.feature]: fixed

File: cypress/fixtures/experimental/checkbox.json (hunk 12 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/checkbox.json]: fixed

File: cypress/fixtures/experimental/checkbox.json (hunk 11 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/checkbox.json]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 10 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 9 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 8 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/fixtures/experimental/checkbox.json (hunk 6 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/checkbox.json]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/fixtures/experimental/checkbox.json (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/checkbox.json]: fixed

File: cypress/fixtures/experimental/checkbox.json (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/checkbox.json]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/features/regression/designSystem/numeralDate.feature [hunk 27]
```diff
@@ -1,22 +1,28 @@
...
       | position | string | result |
       | first    | 123    | 12     |
       | second   | 123    | 12     |
-      | third    | 12345  | 1234   |
\ No newline at end of file
+      | third    | 12345  | 1234   |
+
+  @positive
+  Scenario: Set Numeral Date component date format to <dateFormat>
+    When I open Test test_basic "Numeral Date" component in noIFrame with "numeralDate" json from "test" using "dateFormat1" object name
```
[the remark this answers is not in the document]
Comment (KatarzynaQA) [reply cypress/features/regression/designSystem/numeralDate.feature]: fixed

File: cypress/fixtures/test/numeralDate.json [hunk 28]
```diff
@@ -0,0 +1,5 @@
+{ 
+  "dateFormat1": {
```
[the remark this answers is not in the document]
Comment (KatarzynaQA) [reply cypress/fixtures/test/numeralDate.json]: fixed

File: cypress/support/step-definitions/date-input-steps.js (hunk 26 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/date-input-steps.js]: fixed

File: cypress/features/regression/experimental/checkbox.feature [hunk 29]
```diff
@@ -1,88 +1,84 @@
...
-    When I set label to <label> word
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "<nameOfObject>" object name
     Then checkbox label on preview is <label>
     Examples:
-      | label                        |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | label                   | nameOfObject          |
+      | mp150ú¿¡üßä             | labelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | labelSpecialCharacter |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:11]: why we removed `&"'<>` ?

File: cypress/features/regression/experimental/checkbox.feature [hunk 30]
```diff
@@ -1,88 +1,84 @@
...
-    Then fieldHelp on preview is set to <fieldHelp>
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "<nameOfObject>" object name
+    Then fieldHelp on preview is set to <fieldHelp> in NoIFrame
     Examples:
-      | fieldHelp                    |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | fieldHelp               | nameOfObject              |
+      | mp150ú¿¡üßä             | fieldHelpOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | fieldHelpSpecialCharacter |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:30]: why we removed `&"'<>` ?

File: cypress/features/regression/experimental/checkbox.feature [hunk 31]
```diff
@@ -1,88 +1,84 @@
...
+    When I open default "Experimental Checkbox" component in noIFrame with "checkbox" json from "experimental" using "<nameOfObject>" object name
+      And I hover mouse onto help icon in noIFrame
+    Then tooltipPreview on preview is set to <labelHelp> in NoIFrame
     Examples:
-      | labelHelp                    |
-      | mp150ú¿¡üßä                  |
-      | !@#$%^*()_+-=~[];:.,?{}&"'<> |
+      | labelHelp               | nameOfObject              |
+      | mp150ú¿¡üßä             | labelHelpOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | labelHelpSpecialCharacter |
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/checkbox.feature:60]: why we removed `&"'<>` ?

File: cypress/features/regression/experimental/dateRange.feature [hunk 32]
```diff
@@ -1,42 +1,30 @@
...
+      | label                   | nameOfObject             |
+      | mp150ú¿¡üßä             | endLabelOtherLanguage    |
+      | !@#$%^*()_+-=~[];:.,?{} | endLabelSpecialCharacter |
 
   @positive
   Scenario: Enable labels inline checkbox
-    When I set startLabel to "label"
-      And I set endLabel to "label"
-      And I check labelsInline checkbox
+    When I open default "Experimental Date Range" component in noIFrame with "dateRange" json from "experimental" using "labelsInline" object name
```
Comment (DlgSHi) [inline cypress/features/regression/experimental/dateRange.feature:24]: you have here `labelSinline` and below `labelInlineFalse` in the first variant:
label**S** and in the second label. It should be the same in one component

File: cypress/fixtures/experimental/dateRange.json [hunk 33]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "endLabelSpecialCharacter": {
+    "endLabel": "!@#$%^*()_+-=~[];:.,?{}"
+  },
+  "labelsInline": {
+    "startLabel": "label",
+    "endLabel": "label",
+    "labelsInline": true
+  },
+  "labelInlineFalse": {
```
Comment (DlgSHi) [inline cypress/fixtures/experimental/dateRange.json:19]:
```suggestion
  "labelsInlineFalse": {
```

File: cypress/features/regression/experimental/checkbox.feature (hunk 29 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/features/regression/experimental/checkbox.feature (hunk 31 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/checkbox.feature]: fixed

File: cypress/features/regression/experimental/dateRange.feature (hunk 32 above)
Comment (KatarzynaQA) [reply cypress/features/regression/experimental/dateRange.feature:24]: fixed

File: cypress/fixtures/experimental/dateRange.json (hunk 33 above)
Comment (KatarzynaQA) [reply cypress/fixtures/experimental/dateRange.json]: fixed

**Pull request merged**